docs/sources/flow/reference/components/discovery.marathon.md

```diff
@@ -0,0 +1,141 @@
+---
+canonical: https://grafana.com/docs/agent/latest/flow/reference/components/discovery.marathon/
+title: discovery.marathon
+---
+
+# discovery.marathon
+
+`discovery.marathon` allows you to retrieve scrape targets from [Marathon's](https://mesosphere.github.io/marathon/) Service API.
+
+## Usage
+
+```river
+discovery.marathon "LABEL" {
+  servers = [MARATHON_SERVER1, MARATHON_SERVER2...]
+}
+```
+
+## Arguments
+
+The following arguments are supported:
+
+| Name               | Type           | Description                                                  | Default                 | Required |
+| ------------------ | -------------- | ------------------------------------------------------------ | ----------------------- | -------- |
+| `servers`          | `list(string)` | List of Marathon servers.                                    | `http://localhost:4646` | no       |
```

Comment:
```suggestion
| `servers`          | `list(string)` | List of Marathon servers.                                    |                         | yes      |
```

Reply:
can't believe I made this mistake, thanks!